js/templates/trackerlist.es6.js

```diff
@@ -2,10 +2,10 @@ const bel = require('./../../node_modules/bel');
 
 module.exports = function () {
 
-    return bel`<div class="js-trackerlist js-menu-section">
+    return bel`<div class="js-trackerlist">
         <div class="js-menu-title">${this.model.heading}</div>
         <ul class="js-menu-item-list">
-            ${this.model.companyListM.map((site) => bel`<li> <div class="js-site-blocker-bar" style="width:${site.p}px"></div>
+            ${this.model.companyListM.map((site) => bel`<li class="js-site-co"> <div class="js-site-blocker-bar" style="width:${site.p}px"></div>
```

Comment:
So the list item should have a class to deal with css styling -- only -- as well as the class here that has the prefix `js-` for javascript logic.